src/main/java/org/skriptlang/skript/bukkit/loottables/elements/expressions/ExprLoot.java

```diff
@@ -68,30 +49,24 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 	@Override
 	@Nullable
 	protected ItemStack[] get(Event event) {
-		if (!(event instanceof LootGenerateEvent))
+		if (!(event instanceof LootGenerateEvent lootEvent))
 			return new ItemStack[0];
-		return ((LootGenerateEvent) event).getLoot().toArray(new ItemStack[0]);
+		return lootEvent.getLoot().toArray(new ItemStack[0]);
 	}
 
 	@Override
 	@Nullable
 	public Class<?>[] acceptChange(ChangeMode mode) {
-		switch (mode) {
-			case ADD:
-			case REMOVE:
-			case SET:
-			case DELETE:
-				return CollectionUtils.array(ItemStack[].class);
-			default:
-				return null;
-		}
+		return switch (mode) {
+			case DELETE, ADD, REMOVE, SET -> CollectionUtils.array(ItemStack[].class);
+			default -> null;
+		};
 	}
 
 	@Override
 	public void change(Event event, @Nullable Object[] delta, ChangeMode mode) {
```

Comment:
```suggestion
	public void change(Event event, Object @Nullable [] delta, ChangeMode mode) {
```

Reply:
blame that on copilot it shouldve known to do that after 100 times /j